indigo/distribution.yaml

```diff
@@ -3052,6 +3052,17 @@ repositories:
       url: https://github.com/WPI-RAIL/interactive_world.git
       version: develop
     status: maintained
+  iot_bridge:
+    doc:
+      type: git
+      url: dsd
```

Comment:
WPI-Rail is the previous entry in the rosdistro to your entry it is fine.

Your url and version for documentation are invalid.